### ISSUE TITLE
[White Lion] Pet abilities don't use owner's weapon's DPS

### ISSUE THREAD

**Issue by Blaqwar** [WarEmu/WarBugs#8539]
After the last patch pet damage was changed to use the weapon DPS of the owner. It works for autoattacks but not for abilities. I don't know if this issue is present for other pet careers, the only one I can test it with is an Engineer, I'll try and get those done too.

The screenshots below show a snip of the combat log from a WL pet fighting a lvl 39 mob. First one is with me holding a weapon, the second one is with the character weapon-less. The strength difference doesn't matter since I was using Trained to Threaten stance which gives no offensive stat contribution to the lion. As you can see while auto attack damage is higher with a weapon the damage on Leg Tear/Shred remains the same. (Ignore any small variation in damage since Shred reduces armor and if it get's parried the attacks following it will be getting mitigated, also Shred seems to do damage first and debuff armor second so the damage shred does can vary slightly as well.)

![petdps1](https://cloud.githubusercontent.com/assets/5108812/20868344/c1bfbc70-ba59-11e6-8ca5-93231461ac2e.png)
![petdps2](https://cloud.githubusercontent.com/assets/5108812/20868346/c78add92-ba59-11e6-8a74-e25d6ec99030.png)

I can provide more combat log screenshots if needed but this issue should be very easy to verify.

**Comment by Pxl555** [comment]
Issue seems to affect all pet classes

**Comment by Blaqwar** [comment]
Tested this on my lvl 18 Engineer with Flame Turret. I tried to get as close to the same Ballistic value with the weapons and without as Flame Turret gets 100% of Ballistic stat contribution, I managed to the difference of 2.

With the weapons equipped:
- AA was hitting for 104
- Flamethrower for 183 
- Steam Vents for 59 

Without the weapons the damage was:
- AA for 50
- Flamethrower for 163
- Steam Vents for 55.

It seems there is some scaling but not a lot. Or it might be an oversight on my part but I can't imagine why (I didn't change WS values with the gear changes, nor did I debuff the mobs). Maybe someone with access to better gear options/bigger tooltip values can test as well?
